.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,24 +12,24 @@ This PR adds <github-username> as a contributor.
 
 ## Contributors checklist (check all applicable)
 
-- Have you read the [Getting Started](https://github.com/open-sauced/guestbook#getting-started) section thoroughly?
+### I've read through the [Getting Started](https://github.com/open-sauced/guestbook#getting-started) section.
```

Comment:
@BekahHW we'll remove the Getting Started section as we now have it at the intro course that you've just merged. Do we better use that link instead?


```suggestion
### I've read through the [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) section.
```